data/src/languages.json

```diff
@@ -359,85 +496,130 @@
         "dialect": {
             "us": "US | General American",
             "uk": "UK | Received Pronunciation"
+        },
+        "script": {
+            "zyyy": "Common",
+            "latn": "Latin",
+            "grek": "Greek"
         }
     },
     "epo": {
         "iso639_name": "Esperanto",
         "wiktionary_name": "Esperanto",
         "wiktionary_code": "eo",
-        "casefold": true
+        "casefold": true,
+        "script": {
+            "latn": "Latin"
+        }
     },
     "est": {
         "iso639_name": "Estonian",
         "wiktionary_name": "Estonian",
         "wiktionary_code": "et",
-        "casefold": true
+        "casefold": true,
+        "script": {
+            "latn": "Latin"
+        }
     },
     "ewe": {
         "iso639_name": "Ewe",
         "wiktionary_name": "Ewe",
         "wiktionary_code": "ee",
-        "casefold": true
+        "casefold": true,
+        "script": {
+            "latn": "Latin"
+        }
     },
     "fao": {
         "iso639_name": "Faroese",
         "wiktionary_name": "Faroese",
         "wiktionary_code": "fo",
-        "casefold": true
+        "casefold": true,
+        "script": {
+            "latn": "Latin"
+        }
     },
     "fin": {
         "iso639_name": "Finnish",
         "wiktionary_name": "Finnish",
         "wiktionary_code": "fi",
-        "casefold": true
+        "casefold": true,
+        "script": {
+            "latn": "Latin"
+        }
     },
     "fre": {
         "iso639_name": "French",
         "wiktionary_name": "French",
         "wiktionary_code": "fr",
-        "casefold": true
+        "casefold": true,
+        "script": {
+            "latn": "Latin"
+        }
     },
     "glg": {
         "iso639_name": "Galician",
         "wiktionary_name": "Galician",
         "wiktionary_code": "gl",
-        "casefold": true
+        "casefold": true,
+        "script": {
+            "latn": "Latin"
+        }
     },
     "kld": {
         "iso639_name": "Gamilaraay",
         "wiktionary_name": "Gamilaraay",
         "wiktionary_code": "kld",
-        "casefold": true
+        "casefold": true,
+        "script": {
+            "latn": "Latin"
+        }
     },
     "geo": {
         "iso639_name": "Georgian",
         "wiktionary_name": "Georgian",
         "wiktionary_code": "ka",
-        "casefold": false
+        "casefold": false,
+        "script": {
+            "geor": "Georgian"
+        }
     },
     "ger": {
         "iso639_name": "German",
         "wiktionary_name": "German",
         "wiktionary_code": "de",
-        "casefold": true
+        "casefold": true,
+        "script": {
+            "zyyy": "Common",
```

Comment:
We should double-check which words trigger the "Common" script - it's likely that we need to fold in "Common" script into "Latin" and "Cyrillic" cases.